src/operation_dispatcher.rs

```diff
@@ -347,27 +331,48 @@ mod tests {
         }
     }
 
-    fn build_operation(
-        grpc_call_fn_stub: GrpcCallFn,
-        extension_type: ServiceType,
-    ) -> Rc<Operation> {
+    fn build_auth_grpc_action() -> RuntimeAction {
```

Comment:
Just leaving a note here, for a future refactor... the `operationDispatcher` was meant to be agnostic of the service/extension specifics. Anything specific for auth, ratelimit, what ever else that comes in the future, should be built somewhere else.